src/components/Spinner.tsx

```diff
@@ -1,19 +1,18 @@
-/* @flow */
 import React, { Component } from 'react';
 import { ActivityIndicator } from 'react-native';
 
-import type { Theme } from '../types/Theme';
+import { Theme } from '../types/Theme';
 import { withTheme } from '../';
 
 type Props = {
-  theme: Theme,
-  animating: boolean,
-  size: 'small' | 'large',
-  hidesWhenStopped: boolean,
+  theme: Theme;
+  animating: boolean;
+  size?: 'small' | 'large' | number;
+  hidesWhenStopped: boolean;
 };
 
 class Spinner extends Component<Props> {
-  static defaultProps = {
+  static defaultProps: Partial<Props> = {
```

Comment:
I think this will conflict with inferring default props. I'd do `size: 'small' as const` instead